.pre-commit-config.yaml

```diff
@@ -12,11 +12,13 @@ repos:
       - id: trailing-whitespace
         name: Check trailing whitespace
         args: [--markdown-linebreak-ext=md]
+
```

Comment:
Personal taste, correct also bellow..

Reply:
I would argue it improves readability, but ok.